charts/opencost-config/values.yaml

```diff
@@ -1,3 +1,6 @@
+# -- GKE specific settings
 gke:
+
+# -- If you are using GCP's Managed Prometheus service set this to true
   podmonitoring: false
```

Comment:
With the current setup, should we set this default to true? 
We are currently assuming GKE and GMP to be the default.

Reply:
Its set to true in installation/config/gke/opencost-templates-values.yaml, i.e. the env specific folder for Helm values